src/components/EventListCard/EventListCard.tsx

```diff
@@ -53,6 +55,20 @@ function eventListCard(props: InterfaceEventListCardProps): JSX.Element {
     setEventModalIsOpen(false);
   };
 
+  const showDeleteModal = (): void => {
```

Comment:
create a single toggleDeleteModal function instead of showDeleteModal and hideDeleteModal functions.